syntaxes/architectio.tmLanguage.json

```diff
@@ -5,8 +5,8 @@
 		"Once accepted there, we are happy to receive an update request."
 	],
 	"version": "https://github.com/textmate/yaml.tmbundle/commit/e54ceae3b719506dba7e481a77cea4a8b576ae46",
-	"name": "YAML",
-	"scopeName": "source.yaml",
+	"name": "architectio",
+	"scopeName": "source.architectio",
 	"patterns": [
 		{
 			"include": "#comment"
```

Comment:
what is this file?

Reply:
Language grammars are used to assign names to document elements such as keywords, comments, strings or similar. The purpose of this is to allow styling (syntax highlighting) and to make the text editor “smart” about which context the caret is in. For example you may want a key stroke or tab trigger to act differently depending on the context, or you may want to disable spell check as you type those portions of your text document which are not prose (e.g. HTML tags).

The language grammar is used only to parse the document and assign names to subsets of this document. Then [scope selectors](https://macromates.com/manual/en/scope_selectors) can be used for styling, preferences and deciding how keys and tab triggers should expand.

We want the editor to use the same editing behavior as a yaml file, without it the editor would behave similarly to a plaintext language file.

Reply:
I think the existence of this is another point in favor of using `vscode-yaml` as a dependency rather than the fork-and-modify approach... because the language it's describing is just yaml it's not really something we'd ever modify and would just want to match the underlying `vscode-yaml` version, so changing it to be our own syntax seems like it's adding tech debt that we don't necessarily want to be in the business of maintaining